nusantara/nusa_datasets/id_short_answer_grading/utils/id_short_answer_grading_utils.py

```diff
@@ -0,0 +1,40 @@
+import pandas as pd
+from pathlib import Path
+
+def createSaintekAndSoshumDataset(saintek_question: Path, soshum_question: Path, saintek_score: Path, soshum_score: Path):
+    # creating dataframes
+    saintek_question = pd.read_csv(saintek_question, sep=",", header="infer")
+    saintek_score = pd.read_csv(saintek_score, sep=",", header="infer")
+    soshum_question = pd.read_csv(soshum_question, sep=",", header="infer")
+    soshum_score = pd.read_csv(soshum_score, sep=",", header="infer")
+
+    # renaming
+    saintek_question.rename(columns={'jawaban': 'kunci-jawaban'}, inplace='true')
+    soshum_question.rename(columns={'jawaban': 'kunci-jawaban'}, inplace='true')
```

Comment:
Nit: Is there any particular reason why renaming happens here, and then renamed again on the code below? Should we just do it once?

Reply:
yes it would be correct to say that we should just do it once, i guess it was for when i was testing the code, seeing all columns in the same case feels more pleasing to the eye for me, i'll take care of it right away. also i have change the function names, thank you for the feedback kak!